*** cudf/python/cudf/setup.py-dist	2020-06-14 06:08:09.491573207 +0900
--- cudf/python/cudf/setup.py	2020-06-14 06:14:31.242456439 +0900
***************
*** 31,36 ****
--- 31,37 ----
      raise OSError(f"Invalid CUDA_HOME: directory does not exist: {CUDA_HOME}")
  
  cuda_include_dir = os.path.join(CUDA_HOME, "include")
+ cuda_library_dir = os.path.join(CUDA_HOME, "lib64")
  
  CUDF_ROOT = os.environ.get("CUDF_ROOT", "../../cpp/build/")
  
***************
*** 59,67 ****
          ],
          library_dirs=(
              pa.get_library_dirs()
              + [get_python_lib(), os.path.join(os.sys.prefix, "lib")]
          ),
!         libraries=["cudf"] + pa.get_libraries(),
          language="c++",
          extra_compile_args=["-std=c++14"],
      )
--- 60,69 ----
          ],
          library_dirs=(
              pa.get_library_dirs()
+             + [cuda_library_dir]
              + [get_python_lib(), os.path.join(os.sys.prefix, "lib")]
          ),
!         libraries=["cudf","cuda","cudart","nccl"] + pa.get_libraries(),
          language="c++",
          extra_compile_args=["-std=c++14"],
      )
