*** cudf-dist/python/nvstrings/cpp/CMakeLists.txt	2020-05-04 17:43:11.769790643 +0900
--- cudf/python/nvstrings/cpp/CMakeLists.txt	2020-05-04 17:44:07.166379789 +0900
***************
*** 109,127 ****
  find_library(
      NVSTRINGS_LIBRARY "NVStrings"
      HINTS "$ENV{NVSTRINGS_ROOT}/lib"
!           "$ENV{NVSTRINGS_ROOT}"
  )
  
  find_library(
      NVCATEGORY_LIBRARY "NVCategory"
      HINTS "$ENV{NVSTRINGS_ROOT}/lib"
!           "$ENV{NVSTRINGS_ROOT}"
  )
  
  find_library(
      NVTEXT_LIBRARY "NVText"
      HINTS "$ENV{NVSTRINGS_ROOT}/lib"
!           "$ENV{NVSTRINGS_ROOT}"
  )
  
  message(STATUS "NVSTRINGS: NVSTRINGS_INCLUDE set to ${NVSTRINGS_INCLUDE}")
--- 109,127 ----
  find_library(
      NVSTRINGS_LIBRARY "NVStrings"
      HINTS "$ENV{NVSTRINGS_ROOT}/lib"
!           "../../../cpp/build"
  )
  
  find_library(
      NVCATEGORY_LIBRARY "NVCategory"
      HINTS "$ENV{NVSTRINGS_ROOT}/lib"
!           "../../../cpp/build"
  )
  
  find_library(
      NVTEXT_LIBRARY "NVText"
      HINTS "$ENV{NVSTRINGS_ROOT}/lib"
!           "../../../cpp/build"
  )
  
  message(STATUS "NVSTRINGS: NVSTRINGS_INCLUDE set to ${NVSTRINGS_INCLUDE}")
