*** rmm/python/setup.py-dist	2020-06-14 05:58:58.578386775 +0900
--- rmm/python/setup.py	2020-06-14 06:02:58.393555018 +0900
***************
*** 38,48 ****
              "../include/rmm",
              "../include",
              "../build/include",
              os.path.dirname(sysconfig.get_path("include")),
              cuda_include_dir,
          ],
          library_dirs=[get_python_lib(), os.path.join(os.sys.prefix, "lib")],
!         libraries=["rmm"],
          language="c++",
          extra_compile_args=["-std=c++14"],
      )
--- 38,49 ----
              "../include/rmm",
              "../include",
              "../build/include",
+             "../build/_deps/spdlog-src/include",
              os.path.dirname(sysconfig.get_path("include")),
              cuda_include_dir,
          ],
          library_dirs=[get_python_lib(), os.path.join(os.sys.prefix, "lib")],
!         libraries=["rmm","cudart","nccl"],
          language="c++",
          extra_compile_args=["-std=c++14"],
      )
